src/gfx/context.cpp

```diff
@@ -5,6 +5,7 @@
 #include "platform_surface.hpp"
 #include "window.hpp"
 #include "log.hpp"
+#include <tracy/Tracy.hpp>
```

Comment:
Also guard it with a `#ifdef `

Reply:
Doesn't need to since it does that in the header itself.
It allows you to inserts zones like `ZoneScoped;` without having to wrap those in ifdefs as well